src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -2532,6 +2532,175 @@ void spiral(index_t ndoms,
     }
 }
 
+//---------------------------------------------------------------------------//
+void venn(index_t nx,
+          index_t ny,
+          float64 radius, 
+          Node &res)
+{
+    res.reset();
+    // create a rectilinear coordset 
+    res["coordsets/coords/type"] = "rectilinear";
+    res["coordsets/coords/values/x"] = DataType::float64(nx+1);
+    res["coordsets/coords/values/y"] = DataType::float64(ny+1);
+    
+    float64_array x_coords = res["coordsets/coords/values/x"].value();
+    float64_array y_coords = res["coordsets/coords/values/y"].value(); 
+    
+    // 0 <-> 1
+    float64 dx = 1.0/float64(nx+1);
+    float64 dy = 1.0/float64(ny+1);
+
+    float64 vx = 0;
+    for(index_t i =0; i< nx+1; i++)
+    {
+        x_coords[i] = vx;
+        vx+=dx;
+    }
+    
+    float64 vy = 0;
+    for(index_t i =0; i< ny+1; i++)
+    {
+        y_coords[i] = vy;
+        vy+=dy;
+    }
+    
+    // create the topology
+    
+    res["topologies/topo/type"] = "rectilinear";
+    res["topologies/topo/coordset"] = "coords";
+
+    // create the fields
+
+    // circle a distance field
+    res["fields/radius_a/association"] = "element";
+    res["fields/radius_a/topology"] = "topo";
+    res["fields/radius_a/values"] = DataType::float64(nx * ny);
+    // circle a vf
+    res["fields/circle_a/association"] = "element";
+    res["fields/circle_a/topology"] = "topo";
+    res["fields/circle_a/values"] = DataType::float64(nx * ny);
+
+    // circle b distance field
+    res["fields/radius_b/association"] = "element";
+    res["fields/radius_b/topology"] = "topo";
+    res["fields/radius_b/values"] = DataType::float64(nx * ny);
+    // circle b vf
+    res["fields/circle_b/association"] = "element";
+    res["fields/circle_b/topology"] = "topo";
+    res["fields/circle_b/values"] = DataType::float64(nx * ny);
+
+    // circle c distance field
+    res["fields/radius_c/association"] = "element";
+    res["fields/radius_c/topology"] = "topo";
+    res["fields/radius_c/values"] = DataType::float64(nx * ny);
+    // circle b vf
+    res["fields/circle_c/association"] = "element";
+    res["fields/circle_c/topology"] = "topo";
+    res["fields/circle_c/values"] = DataType::float64(nx * ny);
+
+    // per element how many circles overlap
+    res["fields/overlap/association"] = "element";
+    res["fields/overlap/topology"] = "topo";
+    res["fields/overlap/values"] = DataType::float64(nx * ny);
```

Comment:
Given that all of these fields are initialized in the exact same way, a for-loop over field names (e.g. `radius_*`, `circle_*`, `overlap`) could be used to reduce some of the code duplication.